demos/common/python/models/deblurring.py

```diff
@@ -0,0 +1,102 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import math
+import numpy as np
+
+from .model import Model
+
+class DeblurringModel(Model):
```

Comment:
You should follow @vladimir-dudnik's comment about the name.

Reply:
Ok, I changed the name. But what about `SegmentationModel` and `DetectionWithLandmarks`?

Reply:
They are out of this PR scope any way.